src/config/limits.ts

```
@@ -1,25 +1,177 @@
+    }
+  }
+
+  return result;
+}
+
+export async function getLimitConfig(audience: Audience): Promise<LimitConfig> {
+  // Serve from cache if fresh
+  if (
+    limitsCache &&
+    Date.now() - cacheTimestamp < CACHE_TTL_MS
+  ) {
```

ESLint

[error] Replace `⏎····limitsCache·&&⏎····Date.now()·-·cacheTimestamp·<·CACHE_TTL_MS⏎··` with `limitsCache·&&·Date.now()·-·cacheTimestamp·<·CACHE_TTL_MS`

(prettier/prettier)